reference/sectionA_definitions.inc

```diff
@@ -10,22 +10,29 @@ Jonathan Cooper
 Alan Garny
 Keri Moyle
 David P. Nickerson
+Poul Nielsen
 Hugh Sorby
 
 **Contributors:**
-Jesús C. Fernández
-David Brooks
 Koray Atalag
+David Brooks
+Edmund J. Crampin
+Jesús Carro Fernández
+Peter J. Hunter
+Gary R. Mirams
+Maxwell L. Neal
 
 The authors also wish to acknowledge the significant contribution of the (discontinued) draft CellML 1.2 specification, much of the text of which was incorporated into this specification - although the semantics have changed considerably.
 The CellML 1.2 specification was itself the result of a collaborative effort by a number of researchers during 2008-2011\:
 Andrew K. Miller (who wrote the text reused here),
```

Comment:
Alphabetical order or do we want to make an exception for Andrew?

Reply:
yep, that was the one exception to alphabetical order.

Reply:
I am happy with that. Just wanted to be sure it wasn't an overlook.